libs/auth-api-lib/seeders/data/scope-internal-procuring.ts

```diff
@@ -0,0 +1,17 @@
+import { createScope } from './helpers'
+
+export const up = createScope({
+  // Required:
+  name: '@island.is/internal:procuring',
+  displayName: 'Generic internal API scope for procuring holders',
+  description:
+    'Shared scope for interal usage in cases of delegation for procuring holders',
+
+  // Optional:
+  delegation: {
+    procuringHolders: true,
+  },
+  accessControlled: false,
+  addToResource: '@island.is',
+  addToClients: ['@island.is/clients/application-system'],
```

Comment:
This is probably supposed to be `"@island.is/web"`, which is used in both service-portal and application-system-form.

```suggestion
  addToClients: ['@island.is/web'],
```

Reply:
Ok, the other client seems to exist on DEV though. Maybe some legacy we could remove.

Reply:
It is used on the backend to perform token exchange with client credentials.